data/human/sales.txt

```diff
@@ -102,6 +102,7 @@ shipyard "Northern Pirates"
 	"Dagger"
 	"Firebird"
 	"Headhunter"
+	"Scrapper"
```

Comment:
Alphabetical order.

Reply:
I think you meant to put this on Core Pirates shipyard but alright.